inc/ThirdParty/ServiceProvider.php

```diff
@@ -73,6 +75,9 @@ public function register() {
 		$this->getContainer()->share( 'amp_subscriber', 'WP_Rocket\ThirdParty\Plugins\Optimization\AMP' )
 			->withArgument( $options )
 			->withArgument( $this->getContainer()->get( 'cdn_subscriber' ) );
+		$this->getContainer()->share( 'pressable_subscriber', 'WP_Rocket\ThirdParty\Hostings\Pressable' )
+			->withArgument( $this->getContainer()->get( 'admin_cache_subscriber' ) );
+		$this->getContainer()->share( 'litespeed_subscriber', 'WP_Rocket\Subscriber\Third_Party\Hostings\Litespeed_Subscriber' );
```

Comment:
Shouldn't we also move LiteSpeed in the new structure?
If everybody is ok to keep it like this, then it's fine for me. However I am thinking if things will not mess up in this way. 🤔

Reply:
@crystinutzaa I agree with you. We should. Let's do that as part of a separate PR once this one is merged. Maybe something @engahmeds3ed can work on????

Reply:
@hellofromtonya & @Tabrisrp I approved it 🎉